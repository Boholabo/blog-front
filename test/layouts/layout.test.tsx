import React from 'react';
import { render } from '../testUtils';
import Layout from '../../layouts/layout';

describe('Layout', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <Layout>
        <div>test</div>
      </Layout>,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
