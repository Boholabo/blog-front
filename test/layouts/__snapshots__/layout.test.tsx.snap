// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="theme"
    id="app"
  >
    <div
      id="app-main"
    >
      <header
        class="header"
        id="header"
      >
        <div
          class="header-container container"
        >
          <div
            class="header-header"
          >
            <img
              alt="alt"
              class="header-logo"
            />
            <a
              class="header-link"
              href="/"
              title="$i18n.text.slogan"
            />
          </div>
          <div
            class="header-player"
          >
            <div
              class="panel"
            >
              <button
                class="prev-song button"
              >
                <i
                  class="iconfont icon-music-prev"
                />
              </button>
              <button
                class="toggle-play button"
              />
              <button
                class="next-song button"
              >
                <i
                  class="iconfont icon-music-next"
                />
              </button>
              <button
                class="muted-toggle button"
              >
                <i
                  class="iconfont"
                />
              </button>
            </div>
            <div
              class="song"
            >
              ç©º
            </div>
          </div>
        </div>
      </header>
      <main
        class="main-container"
        id="main"
      >
        <div
          class="main-content"
          id="main-content"
        >
          <div>
            test
          </div>
        </div>
      </main>
      <footer
        class="footer"
        id="footer"
      >
        <div
          class="container"
        >
          <span>
            Powered By
          </span>
          <a
            class="item"
            href="blog"
            rel="noreferrer nofollow noopener"
            target="_blank"
          >
            blog
          </a>
          <span>
            <a
              class="item"
              href="https://boholabo.com"
              rel="noreferrer nofollow noopener"
              target="_blank"
            >
              Boholabo
            </a>
            <a
              href="/about"
            >
              Me
            </a>
          </span>
        </div>
      </footer>
    </div>
  </div>
</DocumentFragment>
`;
